src/popup/stores/SessionStore.ts

```diff
@@ -6,7 +6,7 @@ import { MESSAGE_TYPE, NETWORK_NAMES } from '../../constants';
 import QryNetwork from '../../models/QryNetwork';
 
 const INIT_VALUES = {
-  networkIndex: 1,
+  networkIndex: 0,
```

Comment:
It doesn't actually matter what this value is in the SessionStore, as the value from the networkController will overwrite it. 
I would prefer to only set the value in the networkController, but if I set it to null  or undefined in the  SessionStore the typescript will complain, as it is not a valid number. And if I were to change the type to number | undefined the network would still not be set if the 
`chrome.runtime.sendMessage({ type: MESSAGE_TYPE.GET_NETWORK_INDEX }`
call were to fail.
Do you think we should have it be 0, or do you have any ideas of alternatives?

Reply:
This is fine. It's just a default placeholder either way.